### PR TITLE
Fixes #17: Adds gzip setup by default

### LINKED ISSUE #17: Expose configuration for `gzip` usage

### PR DESCRIPTION
**Pull request by mattrasband**
I pretty much stole [this](https://www.digitalocean.com/community/tutorials/how-to-add-the-gzip-module-to-nginx-on-ubuntu-14-04) to get something basic rolling

### CODE REVIEW

File: templates/etc/nginx/sites-available/site-available.j2 [hunk 1]
```diff
@@ -19,6 +19,14 @@ server {
 }
 
 server {
+    gzip on;
```
Comment (ovv) [inline templates/etc/nginx/sites-available/site-available.j2]: Not sure when you would want to disable `gzip` but maybe we should make a variable for it ?

File: templates/etc/nginx/sites-available/site-available.j2 [hunk 2]
```diff
@@ -19,6 +19,14 @@ server {
...
+    gzip_disable "msie6";
+    gzip_vary on;
+    gzip_comp_level 6;
+    gzip_http_version 1.1;
+    gzip_min_length 256;
+    gzip_types text/plain text/css application/json application/x-javascript text/xml application/xml application/xml+rss text/javascript application/vnd.ms-fontobject application/x-font-ttf font/opentype image/svg+xml image/x-icon;
+
```
Comment (ovv) [inline templates/etc/nginx/sites-available/site-available.j2]: The whole block is situated in the big `ssl` `if`. If needs to be moved down to line 52 https://github.com/pyslackers/ansible-role-nginx/pull/20/files#diff-91bdf8b1c20d6e8c771744d6e40ce195R52

**Review by ovv — CHANGES_REQUESTED** [review]
I restarted one travis job that failed (it happens from time to time with docker and ansible not sure why)

File: templates/etc/nginx/sites-available/site-available.j2 (hunk 2 above)
Comment (mattrasband) [reply templates/etc/nginx/sites-available/site-available.j2]: Good catch, I didn't realize how you set that up. That's a very cool way to do it!

**Review by ovv — APPROVED** [review]

**Pull request merged**